src/lib/atoms/Rating/Rating.tsx

```diff
@@ -0,0 +1,328 @@
+import React, {
+    FC,
+    HTMLAttributes,
+    MouseEvent,
+    ReactNode,
+    cloneElement,
+    isValidElement,
+    useEffect,
+    useLayoutEffect,
+    useState,
+    JSX,
+    KeyboardEvent
+} from 'react';
+import classNames from 'classnames';
+
+// Components
+import SvgSquareIcon from './DefaultSvg';
+
+// Styles
+import './Rating.scss';
+
+const sizes = { small: 32, medium: 36, big: 42 };
+
+export interface IRatingProps {
```

Comment:
We need to extend here from our interface instead of HTMLAttributes<HTMLDivElement> where we should omit some stuff from the existing interface. Please work with @GaroGabrielyan to resolve this comment.

Reply:
We don't need to extend this interface.